doc/integrations.rst

```diff
@@ -1,14 +1,63 @@
 Integrations
 ============
 
+SQLAlchemy
+----------
+
+Install SQLAlchemy 
+
+.. code-block:: python
+
+    pip install sqlalchemy-exasol
+
+
+Connect to Exasol database using SQLAlchemy
```

Comment:
I personally would always write "database" to make clear that it's not about the company. 
But AFAIK Exasol picked the style similar to Oracle, saying simply "Exasol", right? 

```suggestion
Connect to Exasol using SQLAlchemy
```

Reply:
Both is fine I guess, but a general guideline would be nice. 
```suggestion
Connect to Exasol database using SQLAlchemy
```